backend/tests/v5/unit/processors/teamspaces/projects/models/containers.test.js

```diff
@@ -214,6 +218,74 @@ const testGetContainerStats = () => {
 	});
 };
 
+const testAddContainer = () => {
+	describe('Add container', () => {
+		test('should return the container ID on success for teamspace admin', async () => {
```

Comment:
a lot of thesse are N/A as they're just middleware checks (which are already written...)